File: csp-hw/reverseArray.java
public class reverseArray {
    public static void main(String[] args) {
        int[] myArray = { 10, 20, 30, 40 };
        int reversedArray[] = new int[myArray.length];
        int n = myArray.length;
        int i = 0;
        for (i = 0; i < n; i++) {
            reversedArray[i] = myArray[n - 1 - i];
        }
        for (i = 0; i < n; i++) {
            System.out.println(reversedArray[i]);
        }
    }
}

File: csp-hw-python/reverseArray.py
myArray = [10, 20, 30, 40, 50]
reversedArray = [0, 0, 0, 0, 1]
n = len(myArray)
for i in range(0, n):
    reversedArray[i] = myArray[n-i-1]
print(reversedArray)